src/Pages/ProfilePage/ProfilePage.js

```diff
@@ -1,4 +1,216 @@
+import { profile } from '../../data.js';
+import Avatar from '../../assets/images/Avatar.png';
+import themes from '../../Constants/themes.js';
+import { useState, useRef } from 'react/cjs/react.development';
+
+import Button from '../../Components/button/button.js';
+
+let days = [];
+for (let i = 1; i < 32; i++) {
+  days.push(
+    <option key={i} value={i}>
+      {i}
+    </option>
+  );
+}
+
+let years = [];
+for (let i = 1980; i <= new Date().getFullYear(); i++) {
+  years.push(
+    <option key={i} value={i}>
+      {i}
+    </option>
+  );
+}
+
+let months = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+];
+
 function SettingsPage() {
-  return <div>Profile Page</div>;
+  const lang = useRef('');
+  const month = useRef(1);
+  const day = useRef(0);
+  const year = useRef(1980);
+  const [gender, setGender] = useState(profile.gender);
+
+  let imgSrc;
+  profile.Avatar === '' ? (imgSrc = Avatar) : (imgSrc = profile.Avatar);
+  let [theme, setTheme] = useState(profile.theme_id);
+  function onValueChange(event) {
+    setGender(event.target.value);
+  }
+  function saveData() {
+    const Obj = {
+      id: '4ad3fe1b-d179-417d-9927-4b38bbf23e76',
+      email: 'jana.khanji@ieee.org',
+      name: 'Jana Khanji',
+      Avatar: '',
+      language: lang.current.value,
+      gender: gender,
+      birthday: {
+        month: month.current.value,
+        day: day.current.value,
+        year: year.current.value,
+      },
+      theme_id: theme,
+    };
+    console.log(Obj);
```

Comment:
please remove this log

Reply:
the save does not work yet since I did not have the context API working
so I was only logging instead of saving 
I will fix it now